src/Lib.hs

```diff
@@ -205,7 +206,8 @@ processTicket tId = do
     let ticketInfo      = fromMaybe (error "No ticket info") mTicketInfo
     zendeskResponse     <- getZendeskResponses comments attachments ticketInfo
     postTicketComment   <- asksZendeskLayer zlPostTicketComment
-
+   
+    -- What should we do if the zendeskResponse was Nothing?
```

Comment:
The point of this refactoring is to add _error details_ and remove `Maybe` where possible. Maybe you would have an idea what to do with it if you had specific exceptions you could catch here?

Reply:
Ah okay, I'd tackle on that part once I'm done with handling the decompression issue..